blockchain/vm/memory.go

```diff
@@ -133,6 +133,9 @@ func (m *Memory) Print() {
 }
 
 func (m *Memory) Slice(from, to int64) []byte {
+	if from == to {
+		return nil
+	}
```

Comment:
@kjhman21  We need more conditions as you know
```
func (mw *memoryWrapper) slice(begin, end int64) []byte {
	if end == begin {
		return []byte{}
	}
	if end < begin || begin < 0 {
		logger.Warn("Tracer accessed out of bound memory", "offset", begin, "end", end)
		return nil
	}
	if mw.memory.Len() < int(end) {
		// TODO(karalabe): We can't js-throw from Go inside duktape inside Go. The Go
		// runtime goes belly up https://github.com/golang/go/issues/15639.
		logger.Warn("Tracer accessed out of bound memory", "available", mw.memory.Len(), "offset", begin, "size", end-begin)
		return nil
	}
	return mw.memory.GetCopy(begin, end-begin)
}
```

Reply:
Oh, but the return statement is different. I will update this too.